chains/evm/deployment/utils/sequences/sequences.go

```diff
@@ -0,0 +1,14 @@
+package sequences
+
+import (
+	"github.com/smartcontractkit/chainlink-ccip/chains/evm/deployment/utils/operations/call"
+	"github.com/smartcontractkit/chainlink-deployments-framework/datastore"
+)
+
+// OnChainOutput is a standard output type for sequences that deploy contracts on-chain and perform write operations.
+type OnChainOutput struct {
+	// Addresses are the contract addresses that the sequence deployed.
+	Addresses []datastore.AddressRef
+	// Writes are the write operations that the sequence performed.
+	Writes []call.WriteOutput
```

Comment:
call operations can write? I think just a bit confused on the terminologies here